tests/test__bulk_operations.py

```diff
@@ -30,7 +30,7 @@ class BulkOperationsTest(TestCase):
     def setUp(self):
         super(BulkOperationsTest, self).setUp()
         if self.test_with_pymongo:
-            self.client = pymongo.MongoClient()
+            self.client = pymongo.MongoClient(host=os.environ.get('TEST_MONGO_HOST', 'localhost'))
```

Comment:
All of these host definitions are the cost of defining `mongo` as another service in docker-compose: it no longer lives on localhost in this setup.

Benefit: locally, we can define the version we want to use without having to worry about installation gotchas like path or tarball file structure changes.